lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -183,8 +183,8 @@ function dosomething_northstar_build_ns_user($user, $form_state) {
   }
   foreach ($address as $ns_key => $drupal_key) {
     $field = $user->field_address[LANGUAGE_NONE][0];
-    if (isset($field[$drupal_key]['value'])) {
-      $ns_user[$ns_key] = $field[$drupal_key]['value'];
+    if (isset($field[$drupal_key])) {
+      $ns_user[$ns_key] = $field[$drupal_key];
```

Comment:
@chloealee right, sorry my question was a bit unclear
if you save a user profile with an address set, does the entire address with all values still send, I think those were stored in the `value` field

```
19 W. 21 St
NY, NY, USA
```